package.json

```diff
@@ -100,7 +100,8 @@
     "displayName": "OpenShift console monitoring plugin",
     "description": "This plugin adds the monitoring UI to the OpenShift web console",
     "exposedModules": {
-      "MonitoringUI": "./components/alerting"
+      "MonitoringUI": "./components/alerting",
+      "MockLightspeed": "./components/mock-lightspeed"
```

Comment:
```suggestion
```

Reply:
Mock extension used for testing. Removed in latest commit.